variables.tf

```diff
@@ -27,7 +27,13 @@ variable "workspace_alias" {
 variable "alert_manager_definition" {
   description = "The alert manager definition that you want to be applied. See more in the [AWS Docs](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-alert-manager.html)"
   type        = string
-  default     = null
+  default     = <<EOF
```

Comment:
can we use the indented form of heredocs please

```hcl
  default = <<-EOF
	    alertmanager_config: |
	    route:
	      receiver: 'default'
	    receivers:
	      - name: 'default'
  EOF
```

Reply:
Resolved in https://github.com/terraform-aws-modules/terraform-aws-managed-service-prometheus/pull/3/commits/eb016a87545cc56378e3cdbf3042e45bd2ca95cb